components/bandwidth/bandwidth.app.js

```diff
@@ -1,4 +1,59 @@
+const bandwidthMessaging = require("@bandwidth/messaging");
+
 module.exports = {
   type: "app",
   app: "bandwidth",
+  propDefinitions: {
+    messageTo: {
+      type: "string",
+      label: "To",
+      description: "The number the message will be sent to, in E164 format ex `+19195551234`.",
```

Comment:
Technically you don't need periods at the end of these descriptions, since they aren't complete sentences.